api/translate/translate/v1/translate.proto

```diff
@@ -1,5 +1,16 @@
 syntax = "proto3";
 package translate.v1;
+option go_package = "./gen";
```

Comment:
Not required. go package is defined by buf build. @VladislavsPerkanuks , please confirm!

Reply:
It is defined in 
https://github.com/expect-digital/translate/blob/819cceb7ca8249b9e8432defecd0c51af612952a/api/translate/buf.gen.yaml#L4-L5

So this line is redundant